toolbox

```diff
@@ -711,6 +712,18 @@ create()
         monitor_host="--monitor-host"
     fi
 
+    echo "$base_toolbox_command: checking if 'podman create' supports --ulimit host" >&3
+
+    if man podman-create 2>&3 | grep "You can pass 'host'" >/dev/null 2>&3; then
+        echo "$base_toolbox_command: 'podman create' supports --ulimit host" >&3
+
+        ulimit="--ulimit host"
+    else
+        echo "$base_toolbox_command: 'podman create' doesn't support --ulimit host" >&3
+        # shellcheck disable=SC2039
+        ulimit="$--ulimit nofile=$(ulimit -S -n):$(ulimit -H -n)"
```

Comment:
Is that a stray `$`? :)

Given that `podman create --ulimit host` was added right after our baseline Podman version of 1.4.0, I'd rather not implement a fall back path at all. The fall back code paths are usually painful to test, and as a result have a tendency of starting to bit rot. Specifically, in this case, not preserving the ulimits leads to a failure that's specialized enough that those who really care should really get themselves a newer Podman.

Reply:
I dropped the fallback calculation to avoid worrying about the `$`. We can put it back in a subsequent commit if you feel strongly about it.